Drawing/Drawing.js

```diff
@@ -13,20 +13,18 @@
  */
 
 import kind from '@enact/core/kind';
+import EnactPropTypes from '@enact/core/internal/prop-types';
+import ComponentOverride from '@enact/ui/ComponentOverride';
 import {Drawing as UiDrawing} from '@enact/ui/Drawing';
-import Group from '@enact/ui/Group';
 import {Cell, Column, Layout, Row} from '@enact/ui/Layout';
 import PropTypes from 'prop-types';
 import compose from 'ramda/src/compose';
-import {useRef, useState} from 'react';
+import {useEffect, useRef, useState} from 'react';
 
-import BodyText from '../BodyText';
-import Button from '../Button';
+import DrawingControls from './DrawingControls';
+import DrawingUtils from './DrawingUtils';
```

Comment:
these 2 imports should be placed in a separate block, after Skinnable and before css